apps/dev-proxy.config.js

```diff
@@ -1,5 +1,5 @@
-const HOST = process.env.MOLGENIS_APPS_HOST || "https://emx2.dev.molgenis.org";
-const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "FAIR%20data%20hub";
+const HOST = process.env.MOLGENIS_APPS_HOST || "http://localhost:8080";
+const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "pet store";
```

Comment:
don't forget to remove this form PR again

Reply:
Will do, but am gonna leave the pet store change in since the FAIR data hub does not exist.